### ISSUE TITLE
Repairing failed installations / corrupted vagrant boxes

### ISSUE THREAD

**Issue by cosmicvista** [open-learning-exchange/open-learning-exchange.github.io#318]
It is very important to let new interns know that when installing and running vagrant commands on a UNIX based machine such as Linux distributions and Mac OS, to not run commands with root privileges unless it is explicitly required by the terminal. Enabling root privileges when it is not asked for will lead to problems that cannot be fixed such as failure to vagrant up. To remedy this problem, the user will have to delete everything and start from scratch, without issuing root read and write privileges, or they can follow the steps without issuing root privileges unless required. This problem applies to:

"Step #1: BeLL Installation"

AND

"Step #2: Vagrant Tutorial"

**Comment by dogi** [comment]
good thought ;)

when I was writing https://github.com/dogi/ole--vagrant-bells/blob/master/README.md
I thought about mencioning that, and decided to do that implicit ;)
```
sudo apt-get install git
sudo apt-get install virtualbox
sudo apt-get install vagrant
git clone https://github.com/dogi/ole--vagrant-bells.git
cd ole--vagrant-bells/release
vagrant up
```
in other words by using `sudo` I expressed already which user has to be used when running the command ...

maybe we should change the line
`In your Terminal or Command Prompt, type:`
to
`In your Terminal or Command Prompt as normal user, type:`

this is something @aberdean may consider when writing `ole-vagrant-vi` README.md

anyhow non of what I mentioned involves changing something on open-learning-exchange.github.io


We could add this problem to a FAQ page or maybe more generally work on a page which describes how to clean up so somebody could start from scratch ...

@cosmicvista and @aberdean I am really curious in your feedback ;)

**Comment by aberdean** [comment]
@dogi @cosmicvista I don't think it's a good idea to add that to the `ole--vagrant-vi` README.md.

However, it would be useful to add the instructions on how to clean up to faq.md, so if someone happens to make the same mistake, at least s/he can easily find the solution.
Do you think you could write those instructions and submit a pull request, @cosmicvista?

**Comment by cosmicvista** [comment]
ok sure i'll write the instructions.

**Comment by waryhermit** [comment]
@dogi Since @cosmicvista seems to be no longer here, could you clarify what needs to be done for this issue so someone could work on it? Becuase to be honest I wouldn't even know what to do for this one. Or if its relevancy has expired we can close it.

**Comment by dogi** [comment]
We could add this problem to a FAQ page or maybe more generally work on a page which describes how to clean up so somebody could start from scratch ...

**Comment by dogi** [comment]
> However, it would be useful to add the instructions on how to clean up to faq.md, so if someone happens to make the same mistake, at least s/he can easily find the solution.

**Comment by dogi** [comment]
imaging install was only half successful - something went wrong

how would I do - step by step instruction to get my system going

**Comment by dogi** [comment]
@waryhermit think we should also change the title ;)

**Comment by waryhermit** [comment]
@dogi I don't know mac/unix stuff that good, how would you do that? Just the old:

- open git bash
- cd ole--vagrant-vi
- vagrant destroy -f
- cd
- rm -rf ole--vagrant-vi
- git clone https://github.com/dogi/ole--vagrant-vi.git
- cd ole--vagrant-vi
- vagrant up

Or something else?

**Comment by dogi** [comment]
yeah

maybe we should additionally check versions of Git, Vagrant and Virtualbox ....

**Comment by dogi** [comment]
@dogi thinks we should add @waryhermit 's **last** comment to the FAQ pages

Everybody is allowed to do a `pull request` on this issue

**Comment by waryhermit** [comment]
Fixed by #524
